frontend/src/component/UserApp.tsx

```diff
@@ -0,0 +1,14 @@
+import useUser from "../hook/useUser";
+import JobPostingGallery from "./JobPostingGallery";
+
+export default function UserApp() {
+
+    const {user} = useUser("1")
+
+
+    return (<div>
+        {user !== undefined && <JobPostingGallery user={user}/>}
+        {user===undefined&&<p>User is undefined.</p>}
```

Comment:
Tipp: Hier kannst Du einen Ternary-Operator verwenden, um ein "if-else" in React zu nutzen:
```suggestion
        {user ?
            <JobPostingGallery user={user}/>
            :
            <p>User is undefined.</p>
        }
```